sys/include/net/vaina.h

```diff
@@ -0,0 +1,117 @@
+/**
+ * @ingroup     net_vaina
+ * @{
+ *
+ * @file
+ * @brief       VAINA - Versatile Address Interface | Network Administration
+ *
+ * @author      Jean Pierre Dudey <jeandudey@hotmail.com>
+ * @}
+ */
```

Comment:
Use the common file header

Reply:
Solved, added the common file header for both .h and .c files :wink: